build.assets/tooling/cmd/release-notes/template/release-notes.md.tmpl

```diff
@@ -0,0 +1,20 @@
+## Description
+
+{{ .Description }}
+
+## Download
+
+Download the current and previous releases of Teleport at https://goteleport.com/download.
+
+## Plugins
+
+Download the current release of Teleport plugins from the links below.
+* Slack ([Linux amd64](https://cdn.teleport.dev/teleport-access-slack-v{{ .Version }}-linux-amd64-bin.tar.gz))
+* Mattermost ([Linux amd64](https://cdn.teleport.dev/teleport-access-mattermost-v{{ .Version }}-linux-amd64-bin.tar.gz))
+* Discord ([Linux amd64](https://cdn.teleport.dev/teleport-access-discord-v{{ .Version }}-linux-amd64-bin.tar.gz))
+* Terraform Provider ([Linux amd64](https://cdn.teleport.dev/terraform-provider-teleport-v{{ .Version }}-linux-amd64-bin.tar.gz) | [Linux arm64](https://cdn.teleport.dev/terraform-provider-teleport-v{{ .Version }}-linux-arm64-bin.tar.gz) | [macOS amd64](https://cdn.teleport.dev/terraform-provider-teleport-v{{ .Version }}-darwin-amd64-bin.tar.gz) | [macOS arm64](https://cdn.teleport.dev/terraform-provider-teleport-v{{ .Version }}-darwin-arm64-bin.tar.gz) | [macOS universal](https://cdn.teleport.dev/terraform-provider-teleport-v{{ .Version }}-darwin-universal-bin.tar.gz))
+* Event Handler ([Linux amd64](https://cdn.teleport.dev/teleport-event-handler-v{{ .Version }}-linux-amd64-bin.tar.gz) | [macOS amd64](https://cdn.teleport.dev/teleport-event-handler-v{{ .Version }}-darwin-amd64-bin.tar.gz))
+* PagerDuty ([Linux amd64](https://cdn.teleport.dev/teleport-access-pagerduty-v{{ .Version }}-linux-amd64-bin.tar.gz))
+* Jira ([Linux amd64](https://cdn.teleport.dev/teleport-access-jira-v{{ .Version }}-linux-amd64-bin.tar.gz))
+* Email ([Linux amd64](https://cdn.teleport.dev/teleport-access-email-v{{ .Version }}-linux-amd64-bin.tar.gz))
+* Microsoft Teams ([Linux amd64](https://cdn.teleport.dev/teleport-access-msteams-v{{ .Version }}-linux-amd64-bin.tar.gz))
```

Comment:
It might be convenient to build this via Go, given a slice/map/struct/whatever of plugin information. This would make it easier to add new plugins assets in the future.

I've had a few discussions with a couple of teams about adding smoke tests for the docs/release integration. This approach may make adding these tests easier in the future.

Reply:
I considered this but I thought it might be out of scope. I wanted to be more opinionated on how the changelog is parsed and to make it consistent with the releases we have out now for the first iteration. I was worried that structure that I would decide for that plugin information would be too distracting.

But you're right it would be more convenient to have say a package that just contains information on artifacts.